client/stylesheets/_toCat.scss

```diff
@@ -4288,7 +4292,9 @@ On click the images will be displayed at normal size to complete the effect
     width: 75%;
     margin: 5px;
     padding: 15px 10px;
-    background-color: #f8504b;
+    color: $color-spacered;
+    outline: 2px solid red;
```

Comment:
Color `red` should be written in hexadecimal form as `#ff0000`<br>Color literals like `red` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces